samples/data/nl2sql/Readme.md

```diff
@@ -0,0 +1,30 @@
+# Natural Language to SQL Console
+
+`Nl2Sql` provides a sandbox for experimentation and testing of the abilities of LLM's to generate SQL queries based on natural language expression.
+
+[GPT-4 has raised the bar](https://medium.com/querymind/gpt-4s-sql-mastery-2cd1f3dea543) on query generation capabilities.
+
+While other approaches exist in this space, this sample serves to showcase the capability (and limitations) of LLM using [Semantic Kernel](https://github.com/microsoft/semantic-kernel) for *dotnet*.
+Whether or not this approach provides an adequate or cost-effective solution for any particular use-case depends on its specific context and associated expectations.
+
+While a full ecosystem for data-retrieval and processing includes components and capabilities in addition to natural language query generation, this sample aims to:
+
+1. Demonstrate the natural ability of LLM to reason over an objective and generate a SQL query.
+1. Allow exploration of any *(SQL Server)* database (on-premises or cloud hosted).
+
+## Sample Info
+
+The default configuration targets two sample schemas, but it may be configured to target your own database as well.
+
+This sample is organized as follows:
+
+- `nlsql.config` - Contains [data-schemas](./nl2sql.config/schemas/Readme.md) and [semantic-prompts](./nl2sql.config/nl2sql/Readme.md).
```

Comment:
nit: rename to `nl2sql.*` maybe?

Reply:
Just a typo, naming is `nl2sql.*`....fixed.